### PR TITLE
Update gwenhywfar 5.0.0, libchipcard 5.1.5rc2

### PR DESCRIPTION
**Pull request by cstim**

### CODE REVIEW

File: gnucash.modules [hunk 1]
```diff
@@ -147,7 +147,7 @@
 
   <autotools id="gwenhywfar" autogen-sh="configure"
              autogenargs="--with-guis='gtk3' --enable-local-install --disable-binreloc --disable-ssl PKG_CONFIG='pkg-config --dont-define-prefix'">
-    <branch module="223/gwenhywfar-4.99.25rc9.tar.gz" version="4.99.25rc9"
+    <branch module="228/gwenhywfar-5.0.0.tar.gz" version="5.0.0"
```
Comment (fellen) [inline gnucash.modules]: https://www.aquamaniac.de/rdm/attachments/download/231/gwenhywfar-5.1.0.tar.gz is recent.

**Comment by fellen** [comment]
As a spin off from my faltpak update, I have created an improved version of this, too.

**Comment by jralls** [comment]
@fellen, why did you close?

**Comment by fellen** [comment]
Sorry, your approach was obvisious the better choice.

**Pull request closed without merging**